<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionHandleUi</class>
 <widget class="QWidget" name="connectionHandleUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="fileSelectWdg" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSelectFile">
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendFile">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="rightMargin">
       <number>19</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnDisconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="rightMargin">
       <number>19</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QProgressBar" name="lblSendingBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblSending">
        <property name="text">
         <string>Sending File ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="bottomMargin">
         <number>25</number>
        </property>
        <item>
         <widget class="QLabel" name="lblOutput">
          <property name="text">
           <string>Connection Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lstOutput">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
